src/Masa.Blazor/Components/Select/MSelect.cs

```diff
@@ -41,7 +48,7 @@ public class MSelect<TItem, TItemValue, TValue> : MTextField<TValue>, ISelect<TI
 
         [EditorRequired]
         [Parameter]
-        public Func<TItem, TItemValue> ItemValue { get; set; }
+        public Func<TItem, TItemValue>? ItemValue { get; set; }
```

Comment:
The annotation for nullable reference types should only be used in code within a '#nullable' annotations context.